### ISSUE TITLE
QA Ph4: Accessibility - Mobile_Images in social tab are not user friendly with screen reader 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#927]
With Talk Back enabled, the images in the social tab of the modal are not labeled in a user friendly way.  It is difficult to decipher what is being said.  

Steps: 
1. Enable Talk Back in accessibility settings 
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Expand modal > go to the social tab 
5. Tap on the images 
6. Note issue 

Chrome Google Nexus 6 - Android 5.1.1

![gio_927](https://cloud.githubusercontent.com/assets/7585604/15260662/f0b09db8-190c-11e6-89c2-b688317f09c7.png)
